README.md

```diff
@@ -0,0 +1,12 @@
+batch-api
+========
+
+[![Stability: Active](https://masterminds.github.io/stability/active.svg)](https://masterminds.github.io/stability/active.html)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+JSR 352 - Batch Applications for the Java Platform API 1.0
+
+This specification describes the job specification language, Java programming model, and runtime environment for batch applications for the Java platform. It is designed for use on both the Java SE and Java EE platforms. Additionally, it is designed to work with dependency injection (DI) containers without prescribing a particular DI implementation.
+
```

Comment:
Maybe also add a link back to the Eclipse project home with mailing list:
https://projects.eclipse.org/projects/ee4j.batch/contact ?